examples/mobile/HomeApp/js/app.js

```diff
@@ -174,14 +174,22 @@ import Storage from "./clipping-storage.js";
 				input.setAttribute("type", "checkbox");
 				input.setAttribute("id", webClipName);
 
-				label.setAttribute("for", "latest-news-ckeck");
+				label.setAttribute("for", webClipName);
```

Comment:
do we need to set this attribute for?

Reply:
This is the accessibility enhancement. User can tap/click on the label and then set/unset the option. Without this user have to hit the button, which can be small.